rust/riscv64.patch

```diff
@@ -103,26 +101,3 @@ index 99f0b93..56bbbc7 100644
    _pick dest-wasm usr/lib/rustlib/wasm32-*
    _pick dest-src  usr/lib/rustlib/src
  }
-@@ -239,22 +234,6 @@ package_rust() {
-   cp -a dest-rust/* "$pkgdir"
- }
- 
--package_lib32-rust-libs() {
--  pkgdesc="32-bit target and libraries for Rust"
--  depends=(
--    lib32-gcc-libs
--    rust
--  )
--  provides=(lib32-rust)
--  conflicts=(lib32-rust)
--  replaces=(lib32-rust)
--
--  cp -a dest-i686/* "$pkgdir"
--
--  mkdir -p "$pkgdir/usr/share/licenses"
--  ln -s rust "$pkgdir/usr/share/licenses/$pkgname"
--}
--
```

Comment:
why are these removals removed?

Reply:
The `package_$name` functions never gets called if `$pkgname` does not contain `$name`.

Reply:
I think we'd better keep the `PKGBUILD` clean

Reply:
Added back.

Reply:
> I think we'd better keep the `PKGBUILD` clean

I think it would be better to keep a minimal patch for less rotten patch.